src/router/router.js

```diff
@@ -1,11 +1,12 @@
 const openRoutes = require('./routes/open');
 const userRoutes = require('./routes/user');
 const adminRoutes = require('./routes/admin');
+const jwt = require('../shared/jwt');
 
 function init(app) {
     app.use('/', openRoutes);
-    app.use('/u', userRoutes);
-    app.use('/a', adminRoutes)
+    app.use('/', jwt.validateUserToken, userRoutes);
```

Comment:
This makes sense now! Using separate files for different authorizations makes it really convenient here.